### PR TITLE
<feat>(radio): custom radio label

### PR DESCRIPTION
**Pull request by vuuihc**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow DiDi's [contributing guide](https://github.com/didi/cube-ui/blob/master/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: src/components/radio/radio-item.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    value: String,
+    option: {
+      type: [String, Object],
+      required: true
+    },
+    position: {
+      type: String,
+      default: 'left'
+    },
+    horizontal: {
```
Comment (dolymood) [inline src/components/radio/radio-item.vue]: radio-item 应该没有 这个prop

File: src/components/radio/radio.vue [hunk 2]
```diff
@@ -38,32 +39,15 @@
...
-        }
-      },
-      _groupClass() {
-        if (!this.horizontal) {
-          return 'border-top-1px border-bottom-1px'
-        }
-      }
-    },
     watch: {
-      value(newV) {
```
Comment (dolymood) [inline src/components/radio/radio.vue]: 这部分删除了？

File: src/modules/radio/index.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import Radio from '../../components/radio/radio.vue'
+import RadioItem from '../../components/radio/radio-item.vue'
 
 Radio.install = function (Vue) {
   Vue.component(Radio.name, Radio)
+  Vue.component(RadioItem.name, RadioItem)
 }
```
Comment (dolymood) [inline src/modules/radio/index.js]: 新增赋值 Radio.Item = RadioItem

**Comment by dolymood** [comment]
https://github.com/didi/cube-ui/blob/d88f4252c0727c4411ea17a4152af70f441eada8/src/components/radio/radio.vue#L72

https://github.com/didi/cube-ui/blob/d88f4252c0727c4411ea17a4152af70f441eada8/src/components/radio/radio.vue#L64

这两行的变量名不对，应该上次我忘记修改了，顺便也改了吧

另，如果radio要拆的话，checkbox 和 radio 尽量做到统一吧，例如 radio-> radio-group；包括配置项也统一下（label -> option），可以向后兼容下

File: src/components/radio/radio-item.vue (hunk 1 above)
Comment (vuuihc) [reply src/components/radio/radio-item.vue]: horizontal 在label的class的计算中用到了吧

**Pull request merged**